cli/src/main/java/org/jboss/as/cli/gui/OperationDialog.java

```diff
@@ -328,6 +329,10 @@ public RequestProp(String name, ModelNode props, ModelNode value) {
                 this.nillable = props.get("nillable").asBoolean();
             }
 
+            if (props.get("expressions-allowed").isDefined()) {
+                this.expressionsAllowed = props.get("expressions-allowed").asBoolean();
+            }
+
             if (props.get("default").isDefined()) {
```

Comment:
LOL, and of course the wrong style is right here. ;)

Oh, well, even so you might as well make your new code proper.